verifiable-db/src/results_tree/mod.rs

```diff
@@ -1,2 +1,82 @@
 pub(crate) mod binding;
 pub(crate) mod construction;
+/// Old query public inputs, moved here because the circuits in this module still expects
```

Comment:
Given this whole module is unused, I would be more in favor of just removing it all together. We can always go back in time where it was there to copy paste no ? 
That OR importing this module only when enabling a `legacy` feature for example if we want to keep it around live.

Reply:
Added a feature `results_tree` to disable this code until we don't resume development on it